code/__DEFINES/preferences.dm

```diff
@@ -36,7 +36,7 @@
 #define BE_ALIEN		64
 #define BE_PAI			128
 #define BE_CULTIST		256
-#define BE_NINJA		512
```

Comment:
Either that, or something similar to what was done [on -tg.] (https://github.com/tgstation/-tg-station/pull/11803)